requirements.txt

```diff
@@ -0,0 +1,5 @@
+pyqt5
```

Comment:
requirements file is already in repository, incorrectly named doc/dependencies.pip. Thank you for sugestion, I will rename the file, but remove yours please.